src/test/java/com/ncc/aif/ExamplesAndValidationTest.java

```diff
@@ -1485,12 +1537,9 @@ private Model createDiskBasedModel() {
 
     private Model addNamespacesToModel(Model model) {
         // adding namespace prefixes makes the Turtle output more readable
-        model.setNsPrefix("rdf", RDF.uri);
-        model.setNsPrefix("xsd", XSD.getURI());
-        model.setNsPrefix("aida", AidaAnnotationOntology.NAMESPACE);
+        AIFUtils.addStandardNamespaces(model);
```

Comment:
I didn't even know we had this! It looks like this method is the only thing that makes use of it. It would be good to remove the SKOS namespace as none of our stuff uses it and there's no way to make a SKOS object in the AIF library. Should this be a new ticket?

Reply:
Unless you object, I'll just remove SKOS from `AIFUtils.addStandardNamespaces()` as part of this Issue/PR.

Reply:
I was going to suggest that, but I didn't want to feature creep. If you can take care of it in this PR, I'd appreciate it

Reply:
Done.